tests/System.IO.Compression.Tests/BrotliPerfomanceTests.cs

```diff
@@ -138,19 +139,54 @@ public void Decompress(CompressionType type)
         [InlineData(CompressionType.RepeatedSegments)]
         [InlineData(CompressionType.VeryRepetitive)]
         [InlineData(CompressionType.NormalData)]
-        public void Compress(CompressionType type)
+        public void CompressUsingStream(CompressionType type)
         {
             byte[] bytes = CreateBytesToCompress(type);
             foreach (var iteration in Benchmark.Iterations)
             {
                 string filePath = GetTestFilePath();
-                using (FileStream output = File.Create(filePath))
-                using (BrotliStream zip = new BrotliStream(output, CompressionMode.Compress))
+                FileStream output = File.Create(filePath);
+                using (BrotliStream brotliCompressStream = new BrotliStream(output, CompressionMode.Compress))
+                    using (iteration.StartMeasurement())
```

Comment:
Inconsistent use of brace brackets pairs. I would add brace brackets to each scope.